test/stat_tracker_test.rb

```diff
@@ -9,17 +9,20 @@ def setup
     team_path = './fixtures/teams_fixture.csv'
     game_teams_path = './fixtures/game_teams_fixture.csv'
 
-    locations = {
+    file_path_locations = {
       games: game_path,
       teams: team_path,
       game_teams: game_teams_path
     }
 
-    @stat_tracker = StatTracker.from_csv(locations)
+    @stat_tracker = StatTracker.from_csv(file_path_locations)
   end
 
-  def test_it_is_a_class
-    assert_equal Class, @stat_tracker.class
+  def test_it_exists_with_attributes
```

Comment:
This test was changed to reflect this was an instance of StatTracker instead StatTracker being a class. We also tested for the instance variables of the StatTracker.